Dockerfile

```diff
@@ -0,0 +1,129 @@
+FROM ubuntu:16.04
+
+
+LABEL authors="rickard.hammaren@scilifelab.se, phil.ewels@scilifelab.se" \
+    description="Docker image containing all requirements for NGI-RNAfusion pipeline"
+
+#Fusioncatcher 
+RUN apt-get -y update
+RUN apt-get -y install \
+    build-essential\
+    libncurses5-dev \
+    default-jdk \
+    gawk \
+    gcc \
+    g++ \
+    bzip2 \
+    cmake \
+    automake \
+    gzip \
+    zip \
+    unzip \
+    zlib1g-dev \
+    zlib1g \
+    wget \
+    curl \
+    pigz \
+    tar \
+    parallel \
+    libtbb-dev \
+    libtbb2 \
+    python \
+    python-dev \
+    python-numpy \
+    python-biopython \
+    python-xlrd \
+    python-openpyxl
+
+RUN wget http://sf.net/projects/fusioncatcher/files/bootstrap.py -O bootstrap.py && python bootstrap.py -t --download -y && rm bootstrap.py
+
+
+
+#### STAR-Fusion START 
+RUN apt-get update && apt-get install -y gcc g++ perl python automake make \
+                                       wget curl libdb-dev \
+                       bzip2 zlibc zlib1g zlib1g-dev  default-jre \
+                       unzip && \
+    apt-get clean
+
+RUN curl -L https://cpanmin.us | perl - App::cpanminus
+## set up tool config and deployment area:
+
+ENV SRC /usr/local/src
+ENV BIN /usr/local/bin
+
+ENV DATA /usr/local/data
+RUN mkdir $DATA
+
+
+## perl lib installations
+
+RUN cpanm install Set::IntervalTree  # now included w/ STAR-Fusion
```

Comment:
You can do these as a single `RUN cpanm install` command to make the docker build faster:

```bash
RUN cpanm install Set::IntervalTree \  # now included w/ STAR-Fusion
    DB_File \
    URI::Escape \
    Carp::Assert \
    JSON::XS.pm
```